test/core/slice/b64_decode_fuzzer.cc

```diff
@@ -0,0 +1,14 @@
+#include <fuzzer/FuzzedDataProvider.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "src/core/lib/slice/b64.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
+  FuzzedDataProvider data_provider(data, size);
+  const bool url_safe = data_provider.ConsumeBool();
+  const bool multiline = data_provider.ConsumeBool();
+  const auto fuzz_data = data_provider.ConsumeRemainingBytes<char>();
+  grpc_base64_encode(fuzz_data.data(), fuzz_data.size(), url_safe, multiline);
```

Comment:
This is testing base64 encoding but is in a file named "decode".

Reply:
Welp. Oops.